source/codegen/metadata/nixnet/config.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '21.0.0',
+    'c_header': 'nixnet.h',
+    'c_function_prefix': 'nx',
+    'service_class_prefix': 'NiXnet',
+    'java_package': 'com.ni.grpc.xnet',
+    'csharp_namespace': 'NationalInstruments.Grpc.NiXnet',
+    'namespace_component': 'nixnet',
+    'close_function': 'Clear',
+    'code_readiness': 'Release',
+    'driver_name': 'NI-XNET',
+    'status_ok': 'status >= 0',
+    'type_to_grpc_type': {
+        'nxSessionRef_t': 'nidevice_grpc.Session',
+        'nxDatabaseRef_t': 'nidevice_grpc.Session',
+        'nxDatabaseRef_t[]': 'repeated nidevice_grpc.Session',
+        'u32': 'uint32',
+        'u64': 'uint64',
+        'u8': 'uint32',
+        'f64': 'double',
+        'char[]': 'string',
+        'const char[]': 'string',
+        'nxStatus_t': 'int32',
+        'void[]': 'bytes',
+        'nxTimestamp1ns_t': 'uint64',
+        'nxTimestamp100ns_t': 'uint64'
+    },
+    'library_info': {
+        'Linux': {
+            '64bit': {
+                'name': 'nixnet',
+                'type': 'cdll'
+            }
+        },
+        'Windows': {
+            '32bit': {
+                'name': 'nixnet.dll',
+                'type': 'windll'
+            },
+            '64bit': {
+                'name': 'nixnet.dll',
+                'type': 'cdll'
+            }
+        }
+    },
+    'metadata_version': '0.1',
+    'module_name': 'nixnet',
+    'session_class_description': 'An NI-XNET session.',
+    'session_handle_parameter_name': 'vi'
```

Comment:
We don't use this anywhere in codegen yet, but let's keep this variable name to `sessionRef` to match with `nxSessionRef_t` parameters. If we start using this for something, we will probably need to add support `nxDatabaseRef_t` as well here, but for now let's just assume `nxSessionRef_t` is our "main" session and add corresponding name here.

Reply:
I'd change both to `session`. We have to set fields by name in grpc, so these show up in every message and it's nice if it's short. Also `ref`/`handle` are sort of filler and don't map as well/consistently to grpc.

Reply:
Reading through the xnet API, I see a lot of `Ref` param names. And also a lot of different prefixes (interface, db, db_object, etc). If someone disagrees with me on this, I'd love to head a counter-argument.